src/layers/markers/Marker.vue

```diff
@@ -97,7 +102,15 @@
 	set($marker, marker);
 	emit('load', get($marker));
 
-	whenever(position, (position) => get($marker).setLatLng(position));
+	whenever(position, (position) => {
```

Comment:
newPosition, or Position2

Reply:
It is already like that. It doesn't seem troublesome, right?